### DISCUSSION TITLE
Termux remote control [*Enchancement* for *Running on Android*]

### DISCUSSION THREAD

**Discussion by Splarkszter** [ORelio/Minecraft-Console-Client#2173] [Ideas]
Would be cool to add a part that suggests to setup a Remote Controll for Termux for ease of use (Or like my one of a kind case of touch not working because screen died).

Source and Credits: https://joeprevite.com/ssh-termux-from-computer/

Images of it working:

Note: I'm using an APP called Terminal, it's an official clean and useful tool from microsoft if you are interested

![Screenshot (2393)](https://user-images.githubusercontent.com/90066319/189258291-d048146a-0c4b-46a8-a2d0-001038e0ad72.png)
![Screenshot (2394)](https://user-images.githubusercontent.com/90066319/189257985-cf6c10aa-2498-4e06-aa71-1f570c40bad7.png)
![Screenshot (2395)](https://user-images.githubusercontent.com/90066319/189257993-57fa7fb9-8851-4c6a-a488-e4c070e3b564.png)
![Screenshot (2396)](https://user-images.githubusercontent.com/90066319/189258003-c9695303-6836-4a35-99dd-8f543dcacd92.png)

**Comment by milutinke** [comment]
Good idea, I'll add it when I catch some free time
